site/docs/expressions/specialized_record_expressions.md

```diff
@@ -43,7 +43,7 @@ Switch expression allow a selection of alternate branches based on the value of
 ```
 switch(value)
 <value 1> => <return 1> (1 or more times)
-<else> => <return 3>
+<else> => <return n> (the nth return option)
```

Comment:
Do you think this communicates the idea more clearly than the original documentation?